cluster/manifests/s3-csi-driver/daemonset.yaml

```diff
@@ -0,0 +1,180 @@
+        - name: CSI_NODE_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
+        - name: HOST_PLUGIN_DIR
+          value: /opt/podruntime/kubelet/plugins/s3.csi.aws.com/
+        image: container-registry.zalando.net/teapot/aws-mountpoint-s3-csi-driver:v1.9.0-master-21
+        resources:
+          requests:
+            cpu: 1m
+            memory: 50Mi
+            ephemeral-storage: 256Mi
+          limits:
+            cpu: 1m
+            memory: 50Mi
+        livenessProbe:
+          failureThreshold: 5
+          httpGet:
+            path: /healthz
+            port: healthz
+          initialDelaySeconds: 10
+          periodSeconds: 2
+          timeoutSeconds: 3
+        name: s3-plugin
+        ports:
+        - containerPort: 9810
+          name: healthz
+          protocol: TCP
+        securityContext:
+          privileged: false
+        volumeMounts:
+        - mountPath: /opt/podruntime/kubelet
+          name: kubelet-dir
+        - mountPath: /csi
+          name: plugin-dir
+        - mountPath: /run/systemd/private
+          name: systemd-bus
+        - mountPath: /host/dev
+          name: host-dev
+        - mountPath: /host/proc/mounts
+          name: proc-mounts
+      - args:
+        - --csi-address=$(ADDRESS)
+        - --kubelet-registration-path=$(DRIVER_REG_SOCK_PATH)
+        env:
+        - name: ADDRESS
+          value: /csi/csi.sock
+        - name: DRIVER_REG_SOCK_PATH
+          value: /opt/podruntime/kubelet/plugins/s3.csi.aws.com/csi.sock
+        - name: KUBE_NODE_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
+        image: container-registry.zalando.net/teapot/node-driver-registrar:v2.12.0-eks-1-31-4-master-20
+        imagePullPolicy: IfNotPresent
+        name: node-driver-registrar
+        resources:
+          requests:
+            cpu: 1m
+            memory: 50Mi
+            ephemeral-storage: 256Mi
+          limits:
+            cpu: 1m
+            memory: 50Mi
+        volumeMounts:
+        - mountPath: /csi
+          name: plugin-dir
+        - mountPath: /registration
+          name: registration-dir
+      - args:
+        - --csi-address=/csi/csi.sock
+        - --health-port=9810
+        image: container-registry.zalando.net/teapot/livenessprobe:v2.14.0-eks-1-31-4-master-20
+        imagePullPolicy: IfNotPresent
+        name: liveness-probe
+        resources:
+          requests:
+            cpu: 1m
+            memory: 50Mi
+            ephemeral-storage: 256Mi
+          limits:
+            cpu: 1m
+            memory: 50Mi
+        volumeMounts:
+        - mountPath: /csi
+          name: plugin-dir
+      initContainers:
+      - command:
+        - /bin/install-mp
+        env:
+        - name: MOUNTPOINT_INSTALL_DIR
+          value: /target
+        image: container-registry.zalando.net/teapot/aws-mountpoint-s3-csi-driver:v1.9.0-master-21
```

Comment:
One TODO for later maybe would be to refactor this so the image-updater can update it.

Reply:
(same for any other images used)

Reply:
What do you mean? It can update it or?

Reply:
I just misunderstood if bot can update images without the `$image` variables 😅